about.html

```diff
@@ -0,0 +1,142 @@
+    <link rel="stylesheet" href="css/game-img-special_effects.css">
+    <link rel="stylesheet" href="css/background_elements.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>About Microverse Gaming Fest</title>
+</head>
+<body>
+    <main>
+        <!-- Navbar -->
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <!-- Container wrapper -->
+    <div class="container">
+        <!-- Collapsible wrapper -->
+        <div class="collapse navbar-collapse" id="navbarButtonsExample">
+            <ul class="navbar-nav d-flex flex-row">
+                <!-- Icons -->                
+                <li class="nav-item"><a class="nav-link" href="#"><i class="bi bi-facebook"></i></a></li>
+                <li class="nav-item"><a class="nav-link" href="#"><i class="bi bi-twitter"></i></a></li>
+            </ul>
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                <li class="nav-item">
+                    <a class="nav-link" href="#">English</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">MyPage</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <!-- Collapsible wrapper -->
+    </div>
+    <!-- Container wrapper -->
+</nav>
+<!-- Navbar -->
+        <header>
+            <nav class="flex-navbar">
+                <div>
+                    <a href="index.html">
+                        <img class="primary-logo" src="img/MicroverseGamingFestLogo.png" width="10%" alt="MicroverseGamingFestLogo">
+                    </a>
+                </div>
+                <ul class="flex-nav-list">
+                   <li class="list-item" id="homie"><a href="index.html" class="nav-link">Home</a></li>
+                   <li class="list-item"><a href="#about_contact-section" class="nav-link">About</a></li>
+                   <li class="list-item"><a href="#logo-section" class="nav-link">Logo</a></li>
+                   <li class="list-item"><a href="#past_events-section" class="nav-link">Past Events</a></li>
+                </ul>
+                <div class="menu">
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </div>
+            </nav>
+            </header> 
+        <section class="about_contact-section" id="about_contact-section">
```

Comment:
[OPTIONAL]
- [ ] Kinldy for a better touch and feel, you may consider adjusting the height of your `sections` both on the index and about page as you may notice how short they appear to be on your page.